modules/ROOT/pages/monitoring/logging.adoc

```diff
@@ -74,6 +74,16 @@ and xref:configuration/configuration-settings.adoc#config_db.logs.query.threshol
 | label:Windows[]Logs information about errors encountered when installing or running the Windows service.
 |
 |
+
+| _neo4j-admin-exception-trace-DATETIME.log_
+| label:new[Introduced in 2025.01] Logs exceptions that occur during the execution of the `neo4j-admin` commands.
```

Comment:
The `DATETIME` part is formatted as using the Java format string "yyyy-MM-dd.HH.mm.ss" i.e. the full file name would be: `neo4j-admin-exception-trace-2025-01-31.19.31.25.log` Should give an example?